datalad/interface/annotate_paths.py

```diff
@@ -546,9 +546,25 @@ def __call__(
                         **dict(res_kwargs, **path_props))
                     res['status'] = nondataset_path_status
                     res['message'] = 'path not associated with reference dataset'
+                    # TODO: That `path` in following line seems to be incorrect;
+                    # we are operating on `r` or `p` respectively. Guess it
+                    # should be `r`.
+                    # Figure out, whether or not to report on the resolved one:
```

Comment:
Agree. But also cannot tell immediately what is the correct fix.

Reply:
So I'll go for the unresolved one for now, since we probably want to report on the original input. If that turns out to be wrong, we'll notice, I guess. :-)